bazel/bison.bzl

```diff
@@ -1,3 +1,4 @@
+# -*- Python -*-
```

Comment:
Does this have to be "Python" (other files too)?

Reply:
That makes a common editor that doesn't know about bazel files that this is essentially a Python file (which it is).

Reply:
Makes me think.. should we try to add a CI job that checks formatting of bazel files too? We could probably re-use the formatter scripts from sv-test (which use yapf)

(Not as a part of this PR of course)